app/views/spree/users/_open_orders.html.haml

```diff
@@ -13,7 +13,8 @@
         %tr.order-row
           %td.order1
             %a{"ng-href" => "{{::order.path}}", "ng-bind" => "::order.number"}
-          %td.order2{"ng-bind" => "::Orders.shopsByID[order.shop_id].name"}
+          %td.order2
+            %a{"ng-href" => "{{::Orders.shopsByID[order.shop_id].hash}}/shop", "ng-bind" => "::Orders.shopsByID[order.shop_id].name"}
```

Comment:
The URL should be built using the rails routes helpers.
This is angular code so you cannot do:
`main_app.enterprise_shop_path(order.distributor)`
but I think it would be better if you use shop_path instead so:

`"ng-href" => "{{::Orders.shopsByID[order.shop_id].hash}}#{main_app.shop_path}"`

Reply:
I considered that too, but I did not know the right way to do that I angular. Thanks for clarifying.

Reply:
yeah, you just have to know what is ruby and what is angular and remember ruby runs on the server (the #{} part) and then angular (the {{}} part) runs on the browser. Sometimes this creates confusing code like here. I think this is the best we can do here now :+1: